src/ultil.jsx

```diff
@@ -0,0 +1,3 @@
+const availableStatus = ['Not done yet', 'Done', 'In progress'];
```

Comment:
at I mentioned before, STATUS should be declared in a separate file like `config, constants` or another name of `config` file name. 

```js
export STATUSES = {
  CREATED: 'CREATED',
  IN_PROGRESS: 'IN_PROGRESS',
  DONE: 'DONE'
}
```


and show it flexible:

```js
getStatusName(status){
  switch status:
    case STATUSES.CREATED: return 'Chưa làm'
    ...
  
}

```

Reply:
i get it, i just push new commit. please check it when you free. It helps me a lots, thanks you so much :D